.eslintrc

```
@@ -0,0 +1,91 @@
+{
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 8,
+        "ecmaFeatures": {
+            "impliedStrict": true
+        },
+        "sourceType": "module"
```

```suggestion
        "sourceType": "script"
```

@tylercorbett for first week, files need to be linted as type `script`, not `module`. This will fix your eslint error which will fix your travis fail which will fix your PR :)

---

@tylercorbett I believe you can accept this code change directly in GitHub